extensions/csrc/cuda/pybind/scaled_masked_softmax.cpp

```diff
@@ -46,25 +42,13 @@ torch::Tensor bwd(torch::Tensor const& output_grads,
   return bwd_cuda(output_grads, softmax_results, scale_factor);
 }
 
-int get_batch_per_block(int query_seq_len, int key_seq_len, int batches,
-                        int attn_heads) {
-  return get_batch_per_block_cuda(query_seq_len, key_seq_len, batches,
-                                  attn_heads);
-}
-
-}  // end namespace scaled_masked_softmax
-}  // end namespace fused_softmax
-}  // end namespace multihead_attn
-
 PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
-  m.def("forward", &multihead_attn::fused_softmax::scaled_masked_softmax::fwd,
+  m.def("forward", &fwd,
```

Comment:
Would it be better to add a 'scaled_masked_softmax' prefix here?

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/hpcaitech/ColossalAI/assets/28504079/3b068084-996c-49bd-988d-7d80855c6e7b)
Here is the usage, it seems no need to do such things